triage.md

```diff
@@ -0,0 +1,40 @@
+## Table of Contents
+- [Scope](#scope)
+- [Schedule](#schedule)
+- [What Is Triaging](#what-is-triaging)
+- [How to Triage](#how-to-triage)
+
+## Scope
+These guidelines serve as a primary document for triaging issues in [opensearch-benchmark](https://github.com/opensearch-project/opensearch-benchmark) and [opensearch-benchmark-workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads) repositories. For maintainers/contributors its a guide to ensure that we address all customer issues, address feature requests in a timely manner.
+
+## Schedule
+Every week a maintainer will be assigned on a rotational basis to conduct triage meeting.
+The meeting duration will be no more than 1 hour every Tuesday @1.30 PM PDT/3.30 PM CDT
+
+Meeting link: []
+
+## What is Triaging
+Triage is a process of grooming issues with correct labels, assigning them owners if available, ensuring that the issues have all the required information so that they are actionable
+
+## How to Triage
+The below steps are not exhaustive and can be extended. These will be performed for opensearch-benchmark as well as opensearch-benchmark-workloads repositories. In every triage, we should try to cover atleast 10 Issues combined from below categories
+
+- [Untriaged](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aopen+is%3Aissue+no%3Alabel) Issues
+    - Search for Issues which do not have any labels sorted by newest to oldest
+    - Check if there are any duplicates, if yes then comment the duplicate issue link and close one of the two
+
+- [High Priority](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aopen+is%3Aissue+label%3A%22High+Priority%22)
+    - Search for High Priority Issues. Ensure they have a status update or an owner
+
+- [Breaking](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aopen+is%3Aissue+label%3Abreaking)
+    - Search for Issues which are labeled as breaking. Try to assign owners if its high priority beaking change
+
+- [Bugs](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aissue+is%3Aopen+label%3Abug)
+    - Search for Issues which are labeled as bugs
+    - Provide status update such as In Progress, Not Planned, Need more information.
+
+- [Enhancements](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-desc+label%3Aenhancement+)
+    - Search for feature requests/enhancements. Provide status update and assign owners if its going to be next release candidate
+
+
+*There could be overlap of an Issue in more than 1 category such as Breaking and Bugs or Breaking and High Priority. In this case triage only once and pick another issue. Also ensure that*
```

Comment:
Is there a sentence missing at the end? "Also ensure that"

Reply:
Let me update